packages/components/src/internal/util/global.ts

```diff
@@ -13,34 +13,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { getGlobal, setGlobal } from 'reactn';
+// import { getGlobal, setGlobal } from 'reactn';
 import createHistory from 'history/createBrowserHistory';
 
+let _BrowserHistory;
+
 /**
  * Initialize the global state object for this package.
  */
 export function initBrowserHistoryState() {
```

Comment:
I think we should get rid of this function and just make the line above look like:

```ts
const _BrowserHistory =  createHistory();
```

This would remove the need to ever call initQueryGridState in tests, and could remove the need to throw an error below.

Reply:
I like this suggestion, but after setting `_BrowserHistory` directly to `createHistory()`, I get a type error in util/URL.ts. So I'm going to leave this as is for now.
<img width="579" alt="Screen Shot 2022-07-13 at 3 54 21 PM" src="https://user-images.githubusercontent.com/6411206/178832623-8029724b-00d5-40ff-8533-8ab49275601f.png">

Reply:
Fine to leave it as is for now, but we really should figure out why that is happening. Some type somewhere is wrong, and that needs to be fixed. We may need to update the library we use, or the types library associated with it.